src/components/product-details/bid-tab/index.tsx

```diff
@@ -3,18 +3,164 @@ import TabContainer from 'react-bootstrap/TabContainer';
 import TabContent from 'react-bootstrap/TabContent';
 import TabPane from 'react-bootstrap/TabPane';
 import Nav from 'react-bootstrap/Nav';
-import { Author, History, Property, Tag } from '@types';
+import { Author, History, Property, Tag, Image as ImageType } from '@types';
 import DetailsTabContent from './details-tab-content';
 import HistoryTabContent from './history-tab-content';
+import Img from '../../../../public/favicon.png';
+import Img2 from '../../../../public/images/brand/brand-01.png';
+import Img3 from '../../../../public/images/brand/brand-02.png';
+import Img4 from '../../../../public/images/brand/brand-03.png';
+import BoyAvatar from '../../../../public/images/icons/boy-avater.png';
 
 type Props = {
   className?: string;
-  owner: Author;
-  properties: Property[];
-  tags: Tag[];
-  history: History[];
+  owner?: Author;
+  properties?: Property[];
+  tags?: Tag[];
+  history?: History[];
 };
 
+const historyHard: any = [
```

Comment:
Put a TODO comment that this hardcoded data is for demo purposes and must be removed so we at least have some way to track this work